commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/UriParserTest.java

```diff
@@ -99,4 +100,20 @@ public void testTypeOfNormalizedPath() {
             fail(e);
         }
     }
+
+    @Test
+    public void testPathOfNormalizedPath() throws FileSystemException {
+        checkNormalizedPath("./Sub Folder/", "/Sub Folder");
```

Comment:
I think this PR should only address ONE claim because I am not sure this is correct according to [RFC 3986 Section 3.3](https://www.rfc-editor.org/rfc/rfc3986#section-3.3) regarding trailing "/": A path "/foo/bar" has two segments and a path "/foo/bar/" has 3 segments where the last segment is empty. At least that's how I read it. I'll also [ask](https://lists.apache.org/thread/n4nrod16m7lvs36n0bcdh9w587gg8h8d) on the [dev mailing list](https://commons.apache.org/mail-lists.html), where you are welcome to participate as well.

Reply:
Hi Gary, 
 Yes, this PR should address one claim. This PR doesn't change the behavior regarding the trailing "/". This is what's already happening on master. It's just that as I was writing unit tests, I was thinking to add tests regarding what's currently happen for the "/".
 Let me know what you prefer:
* Remove the unit tests regarding the trailing "/"
* Remove all the unit tests regarding "/"
* Create a new issue that normalisePath should keep the trailing "/"